controllers/topic-controller.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const request = require("express");
+const fetchTopics = require("../models/topic-model");
+
+function getTopics(request, response, next) {
+  fetchTopics()
+    .then((topics) => {
+      response.status(200).send({ topics });
+    })
+    .catch((err) => {
+      console.log(err, "ERROR IN THE CONTROLLER");
+      next(err);
+    });
```

Comment:
Don't forget to clear out any console logs once you have finished development 🧹  if console logs go to main they will continue to log every time this controller's catch is executed